### PR TITLE
chore(docs): Update systemClockOffset example

### PR DESCRIPTION
**Pull request by esauerbo**
#### Description of changes
- Update to show max clock skew greater than or equal to 5 min
- Update example to use mock server time and add note about getting server time  

#### Checklist

- [x] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [x] PR description included
- [x] PR title and commit messages follow [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) syntax

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by timngyn — DISMISSED** [review]

File: docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx [hunk 1]
```diff
@@ -14,25 +14,30 @@ function MyComponent() {
   const [createLivenessApiData, setCreateLivenessApiData] = React.useState(null);
   
   /* 
-   * 1. Check whether the difference between server time and device time
-   * is greater than 5 minutes, and if so, return the offset in milliseconds
-   * This logic should be adjusted based on the server response and your use case
+   * 1. Check whether the difference between server time and device time is
+   *  greater than or equal to 5 minutes, and if so, return the offset in milliseconds.
+   *  This logic should be adjusted based on the server response and your use case
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx:19]:
```suggestion
   *  This logic should be adjusted based on the server response and use case
```

**Review by timngyn — APPROVED** [review]

File: docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx [hunk 2]
```diff
@@ -14,25 +14,30 @@ function MyComponent() {
...
-    * This should be replaced with your own API call to create a session
+    * Replace with your own API call to create a session
     */
     const response = await fetch(`/api/createSession`);
     const body = await response.json(); // { sessionId: 'mockSessionId' } 
-    const systemClockOffset = getSystemClockOffset(response.headers['date'])
-    setCreateLivenessApiData(body, systemClockOffset);
+    /*
+    * Replace serverDate with the actual server date,
+    * which can be retrieved from the response headers or your customer backend.
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx:36]:
```suggestion
    * which can be retrieved from the response headers or your custom backend.
```

File: docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx [hunk 3]
```diff
@@ -14,25 +14,30 @@ function MyComponent() {
...
+    */
+    const serverTime = Date.now();
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx:38]: We should probably show default example of `const serverTime = response.headers['date']`, right?

File: docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx [hunk 4]
```diff
@@ -14,25 +14,30 @@ function MyComponent() {
...
     /*
-    * This should be replaced with your own API call to create a session
+    * Replace with your own API call to create a session
     */
     const response = await fetch(`/api/createSession`);
     const body = await response.json(); // { sessionId: 'mockSessionId' } 
-    const systemClockOffset = getSystemClockOffset(response.headers['date'])
-    setCreateLivenessApiData(body, systemClockOffset);
+    /*
+    * Replace serverDate with the actual server date,
```
Comment (reesscot) [inline docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx:35]:
```suggestion
    * Replace serverTime with the actual server date,
```

File: docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx (hunk 3 above)
Comment (esauerbo) [reply docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx:38]: Yeah mstm

File: docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx [hunk 5]
```diff
@@ -14,25 +14,30 @@ function MyComponent() {
...
+    * which can be retrieved from the response headers or your custom backend.
+    */
+    const serverTime = Date.now();
```
Comment (esauerbo) [inline docs/src/pages/[platform]/connected-components/liveness/customization/customization.system-clock-offset.react.mdx:38]:
```suggestion
    const serverTime = response.headers['date']
```

**Review by reesscot — APPROVED** [review]

**Review by timngyn — APPROVED** [review]

**Pull request merged**